### PR TITLE
My first microverse project

### PR DESCRIPTION
**Pull request by 0zedd**
Added color to style the welcome message in this feature

### CODE REVIEW

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello @0zedd  😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  No linter errors

## Required Changes ♻️

- [ ] Kindly add some description in the Pull Request like what you have done in this Pull Request.
- [ ] Please add a meaningful title for this Pull Request. 
- [x] As you can see, you created the revert-3-main branch from the feature branch. The branch should be created from the main branch. 
- [x] There are no files committed to this branch. In the requirements, You should commit at least 2 files (1. index.html and style.css file) in this branch.


_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.css [hunk 1]
```diff
@@ -1 +1 @@
-h1 { color: purple; }
+h1 { color: blue; }
```
Comment (vechicin) [inline index.css:1]: [OPTIONAL] Using color names is considered bad practice, I recommend that, moving forward, you use the `rgb` or `hex` codes when applying colors. I suggest you change this color to its `rgb` or `hex` equivalent.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @0zedd  ✋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🚨 


 ## Highlights ⚡ 

- [x] Good job implementing some of the changes required by the previous reviewer!
- [x] Linters checks passing
- [x] Good PR title
- [x] Correct GitHub Flow 

 ## Required Changes ♻️

- [ ] I recommend you change your PR description. I don't need to know the specific details of what you did in the feature, just a general overview. Here's an example for you:
In this feature, I added a header with the `Hello Microverse!` text and styled it with my favorite color.

_Check the comments under the review._

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by vechicin** [comment]
https://github.com/0zedd/Linters/blob/e2af5ac5602cca628fb8e10ab214f7965e91eb15/README.md?plain=1#L1-L80
[OPTIONAL] Your README needs some tweaks, especially in the `Description`, `Getting Started`, and `Author` sections. The previous reviewer accidentally missed checking your README, but if this was the first review I would request these changes as mandatory so I strongly suggest you follow my suggestions for this file.

For the `Description`, just add a brief description of what this project is about. The `Getting Started` section should have instructions on how to get a copy of this project locally and have it run properly. The `Author` section should have however many authors worked in the project with their actual names as opposed to `Author1` titles.

I also recommend you capitalize the technologies you use, having them lowercased makes your documentation, and therefore, your project, look unprofessional.

**Comment by vechicin** [comment]
https://github.com/0zedd/Linters/blob/1b3adcf8b84ba192e486f6dbc90d92c4298111c9/test.md#L1
The comments above apply to this file as well.

**Comment by vechicin** [comment]
https://github.com/0zedd/Linters/blob/1b3adcf8b84ba192e486f6dbc90d92c4298111c9/.node_modules/.hintrc#L1-L18
I recommend you copy both the `.hintrc` and the `stylelintrc.json` files to the **root directory** of your project, not the `node_modules` folder, otherwise, these files will be untracked and you will cause GitHub Actions to fail.

**Review by Kakalanp — APPROVED** [review]
# STATUS: APPROVED :green_circle: 

### Your project is approved @0zedd ! :tada: 

There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :clinking_glasses: 

almost there

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**